bin/bundle

```diff
@@ -14,7 +14,7 @@ m = Module.new do
   module_function
```

Comment:
I have questions about this file

Reply:
part of this is from bundle update/install the rest is formatting